zproject/computed_settings.py

```diff
@@ -163,9 +164,9 @@
 
 
 class TwoFactorLoader(app_directories.Loader):
-    def get_dirs(self) -> List[str]:
+    def get_dirs(self) -> List[PosixPath]:
         dirs = super().get_dirs()
-        return [d for d in dirs if 'two_factor' in d]
+        return [d for d in dirs if d.match("two_factor/*")]
```

Comment:
Could we just provide a fixed path here instead?

Reply:
The path in dev env is `/srv/zulip-py3-venv/lib/python3.6/site-packages/two_factor/templates`, so I think the best we can do here that will work across various installations would be grabbing the path from `two_factor.__file__` - but is that better than the current way?

Reply:
Hmm, I see; yeah I don't have an obvious answer that's not about as ugly as what we have here.